### ISSUE TITLE
[FEAT] Update informer

### ISSUE THREAD

**Issue by skyf0l** [GoToolSharing/htb-cli#83]
As there are often new versions of `htb-cli`, I'd like to be notified by launching the CLI to install the update. See how it looks with [RsaCracker](https://github.com/skyf0l/RsaCracker) (with the [update_informer](https://crates.io/crates/update_informer) crate): 

```
$ rsacracker 
A new release of rsacracker is available: v0.6.3 -> v0.6.4
You can update by running: cargo install rsacracker
```

I think it would be nice to have a one-line update that would also update autocompletion.

**Comment by QU35T-code** [comment]
```bash
> go run . stop
A new update is now available (dev) ! (ece0cda2365f65195470102e20a693fe8b22a3a5)
Update with : git pull

No machine is running
```

**Comment by QU35T-code** [comment]
Added
